### PR TITLE
MattTytel.Helm version 0.9.0.0

### PR DESCRIPTION
**Pull request by 25letter**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/50175)

### CODE REVIEW

File: manifests/m/MattTytel/Helm/0.9.0/MattTytel.Helm.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: MattTytel.Helm
+PackageVersion: 0.9.0
```
Comment (Trenly) [inline manifests/m/MattTytel/Helm/0.9.0/MattTytel.Helm.installer.yaml:5]:
```suggestion
PackageVersion: 0.9.0.0
```
This must match control panel

File: manifests/m/MattTytel/Helm/0.9.0/MattTytel.Helm.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: MattTytel.Helm
+PackageVersion: 0.9.0
```
Comment (Trenly) [inline manifests/m/MattTytel/Helm/0.9.0/MattTytel.Helm.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 0.9.0.0
```

File: manifests/m/MattTytel/Helm/0.9.0/MattTytel.Helm.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: MattTytel.Helm
+PackageVersion: 0.9.0
```
Comment (Trenly) [inline manifests/m/MattTytel/Helm/0.9.0/MattTytel.Helm.yaml:5]:
```suggestion
PackageVersion: 0.9.0.0
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You need to update the version folder as well.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**